compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -351,7 +351,7 @@ TR::Instruction *fixedSeqMemAccess(TR::CodeGenerator *cg, TR::Node *node, intptr
    else
       {
       TR::MemoryReference *memRef = new (cg->trHeapMemory()) TR::MemoryReference(baseReg, ((int32_t)loAddr)<<16>>16, opSize, cg);
-      if (op.isLoad())
+      if (!op.isStore())
```

Comment:
Just curious why this was needed.

Reply:
The full explanation is in the description of the relevant commit (8b19964068a8591266d40962244c2448d88453e6), but the TL;DR is that `addi` is neither a load nor a store but needs to be treated as if it were a load here.